core/chains/evm/logpoller/log_poller.go

```diff
@@ -71,8 +71,9 @@ type Client interface {
 
 var (
 	_                          LogPollerTest = &logPoller{}
-	ErrReplayAbortedByClient                 = errors.New("replay aborted by client")
-	ErrReplayAbortedOnShutdown               = errors.New("replay aborted, log poller shutdown")
+	ErrReplayAbortedByClient                 = errors.New("client aborted, replay request cancelled")
```

Comment:
Who/what is the `client`?

Reply:
If 'client' includes anyone/anything upstream that is able to cancel that particular context, then I don't think it adds value to say. The request was simply cancelled. The only reliable way to interpret it is via the new `context.Cause` method https://pkg.go.dev/context#Cause

Reply:
I agree "client" is somewhat vague.  This has been in there since the beginning of logpoller, the only purpose of it is to distinguish it from a shutdown event where it was aborted because logpoller is shutting down.  Are you saying we shouldn't distinguish at all between the two?  Or just that we should remove the word "client" from the error codes and messages, since it doesn't add much and could mean almost anything?  How about `ErrReplayAborted`, `ErrLogPollerShutdown`, and `ErrReplayInProgress`?

Reply:
That sounds good.

Reply:
Actually, I think `ErrReplayRequestAborted` is better, since `ErrReplayAborted` makes it sound as if the replay has already started.